app/src/main/res/layout/controller_generic_rv.xml

```diff
@@ -28,7 +29,7 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/recycler_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
-    </android.support.v7.widget.RecyclerView>
+        android:layout_height="match_parent"
+        tools:listitem="@layout/rv_item_call"/>
```

Comment:
Unsure if you can do this, since tis layout also uses rv_item_contact, and will use others in the future?

Reply:
This can be done ;) the ```tools:listItem``` attribute, like the ```tools:text``` is for Android Studio only for the preview rendering and has no impact at run-time, see https://developer.android.com/studio/write/tool-attributes.html

Reply:
Awesome. Never cared much about the previews :P Thanks for the explanation.